packages/client/txGenerator.ts

```diff
@@ -0,0 +1,186 @@
+// Adapted from - https://github.com/Inphi/eip4844-interop/blob/master/blob_tx_generator/blob.js
+import { randomBytes } from 'crypto'
+import { Address } from '@ethereumjs/util'
+import { Client } from 'jayson/promise'
+import { BlobEIP4844Transaction } from '@ethereumjs/tx'
+import {
+  blobsToCommitments,
+  commitmentsToVersionedHashes,
+} from '@ethereumjs/tx/test/utils/blobHelpers'
+import { loadTrustedSetup, freeTrustedSetup } from 'c-kzg'
+const input = process.argv[2]
+const expected_kzgs = process.argv[3]
+
+const BYTES_PER_FIELD_ELEMENT = 32
+const FIELD_ELEMENTS_PER_BLOB = 4096
```

Comment:
we need to parametrize this thing as for CL/lodestar tests which run in minimal setting the size could mismatch: https://github.com/ethereum/consensus-specs/blob/dev/presets/minimal/eip4844.yaml

Reply:
That's interesting.  I think those two constants are defined by the 4844 spec.  I haven't landed on a final way that we're going to put all of this stuff together but I'm more and more going to either a specific export in the `util` library where all the constants and helpers live or else an entirely separate package for blob stuff like @holgerd77 mentioned at one point.